ui/src/main/webapp/components/ControlWrapper.tsx

```diff
@@ -114,6 +153,8 @@ class ControlWrapper extends React.PureComponent {
                     help={helpText}
                     tooltip={tooltip}
                     error={this.props.error}
+                    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                    // @ts-ignore property should be data-name, but is mapped in obj ControlGroupWrapper
```

Comment:
I did not get, what the exact problem to set proper attribute?

Reply:
ah, I got it. I don't understand why we have mapping instead of providing `data-name` directly